packages/services/src/services/Analytics/Analytics.js

```diff
@@ -0,0 +1,78 @@
+import root from 'window-or-global';
+
+/**
+ * @constant {boolean} scrollTracker determines whether scroll tracking analytics is enabled
+ * @private
+ */
+const scrollTracker = process.env.SCROLL_TRACKING === 'true' || false;
```

Comment:
I'm not sure I understand why this was added. Why are we allowing them to opt into something like this with an extra step at that? I feel like this is adding complexity to the adopter's UX unnecessarily. I also feel like scroll tracking is something we want on every page, and leaving this door open for every adopter to opt-in doesn't sound like a great strategy for getting that data. Maybe I'm not aligned on the intent here though?

Reply:
This was something Juan wanted as well - 

Juan: "that will only be enabled on-demand, meaning it is off by default and will be enabled with an option or config function"

Reply:
Ok, cool. I feel like this hurts the quality of our analytics, but ok. It is what it is I guess. I feel like Juan should be doing this work. 😉haha